<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientTCPIPClass</class>
 <widget class="QMainWindow" name="ClientTCPIPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientTCPIP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="tempGroup">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Température</string>
    </property>
    <widget class="QComboBox" name="tempCombo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Température en °C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Température en °F</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hygrométrie</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="tempLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Récupérer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sensorLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Capteur</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="getButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Récupérer</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="dataBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>bananes</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="dataLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Données récupérées</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sensorLine">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="connectGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Connexion</string>
    </property>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ipLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse IP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipLine">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="portLine">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="connectStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Etat connexion : Déconnecté</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ClientTCPIP.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>ClientTCPIPClass</receiver>
   <slot>OnConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>connected()</signal>
  <signal>disconnected()</signal>
  <signal>readyRead()</signal>
  <slot>OnConnectButtonClicked()</slot>
  <slot>OnSocketConnected()</slot>
  <slot>OnSocketDisconnected()</slot>
 </slots>
</ui>
